src/azul/plugins/metadata/hca/transform.py

```diff
@@ -82,6 +86,126 @@
 pass_thru_uuid4: PassThrough[api.UUID4] = PassThrough()
 
 
+class OrganismAge(FieldType[JSON, str]):
+
+    def to_index(self, value: Optional[JSON]) -> str:
+        """
+        >>> a = OrganismAge()
+        >>> a.to_index({'value': '20', 'unit': 'year'})
+        '20 year'
+
+        >>> a = OrganismAge()
+        >>> a.to_index({'value': '20', 'unit': None})
+        '20'
+
+        >>> a = OrganismAge()
+        >>> a.to_index({'value': None, 'unit': None})
+        '~null'
+
+        >>> a.to_index(None)
+        '~null'
```

Comment:
These two cases highlight the ambiguity. Two different value are mapped to the null string. See if you can eliminate one, preferably the more complex one.